cmake/FindDependencies.cmake

```diff
@@ -42,8 +42,10 @@ ENDIF()
 
 IF(USE_PNG)
 	IF(APPLE AND DEFINED ENV{GITHUB_SHA})
-		SET(PNG_INCLUDE_DIRS "/usr/local/Cellar/libpng/1.6.40/include")
-		SET(PNG_LIBRARIES "/usr/local/Cellar/libpng/1.6.40/lib/libpng16.16.dylib")
+		#brew --cellar libpng -> /usr/local/Cellar/libpng
+		#brew list --versions libpng -> libpng 1.6.42
+		SET(PNG_INCLUDE_DIRS "/usr/local/Cellar/libpng/1.6.42/include")
+		SET(PNG_LIBRARIES "/usr/local/Cellar/libpng/1.6.42/lib/libpng16.16.dylib")
```

Comment:
This way we're just kicking the ball down the line. Ideally we could just use globbing, but I'm sceptical that would work.

find_path can be used to locate the dylib, since that name doesn't change:
https://cmake.org/cmake/help/latest/command/find_path.html
(I suggest using NO_DEFAULT_PATH for greater robustness)

... and then you can append "../../include" and bam, we also have the value for the include dir.

Reply:
I failed making it with `find_path`. It is not so obvious to me and hard to test without macOS locally.